otx/algorithms/classification/utils/data.py

```diff
@@ -115,6 +120,15 @@ def __init__(
                 dataset_item = DatasetItemEntity(image, annotation_scene, subset=subset)
                 dataset_items.append(dataset_item)
 
+        if ul_data_root is not None:
```

Comment:
How about `if ul_data_root:`

Reply:
"is not None" is better in readability. and it's also faster than just "if ul_data_roots"